### DISCUSSION TITLE
Is this working on Multi farms MP Dedi server ?

### DISCUSSION THREAD

**Discussion by Lactic68** [Royal-Modding/FS19_CallContractors#8] [Ideas]
@TyKonKet 
Hi,
Congratulations for this mod ! This is going to be a real help for small farms with lot of fields.

I have a question : 
Is it working like a contracts marketplace in a multiplayer environment ?
Example : Farm A is looking for a contractor to lime field 30. He publish his request and then wait for a contractor to take the job. The contractor might be Farm B (other players in the same map).

**Comment by TyKonKet** [comment]
No, at the moment the contractors are only virtual NPCs.

**Reply by Lactic68** [reply]
Ok, thank you for your quick answer
